app/assets/javascripts/components/components/status_content.jsx

```diff
@@ -39,7 +40,7 @@ const StatusContent = React.createClass({
       } else if (link.textContent[0] === '#' || (link.previousSibling && link.previousSibling.textContent && link.previousSibling.textContent[link.previousSibling.textContent.length - 1] === '#')) {
         link.addEventListener('click', this.onHashtagClick.bind(this, link.text), false);
       } else if (media) {
-        link.innerHTML = '<i class="fa fa-fw fa-photo"></i>';
+        link.innerHTML = '<Icon icon="photo" />';
```

Comment:
went back to the static HTML string in https://github.com/tootsuite/mastodon/pull/1562/commits/b18d4752cd033322811c839462953a7304c67a4d

so icon code isn't 100% DRY 😞 due to the one spot `.innerHTML` is used